File: Node.JS/src/models/Customer.js
//customer model for populating customer testing data
const mongoose = require('mongoose')

const customerSchema = mongoose.Schema({
    firstName:{
        type:String, 
        required: true
    }, 
    lastName:{
        type:String, 
        required: true
    }, 
    age:{
        type:String, 
        required: true
    }, 
    orders: [{
       type: mongoose.Schema.Types.ObjectId,
       ref : "Order"
    }], 
    address : {
        type: mongoose.Schema.Types.ObjectId,
        ref : "Address"
    }, 
   phoneNumber: {
       type:String, 
       required: true
   },
   join_date:{
       type:Date,
       default:Date.now
   }
})
module.exports = mongoose.model('Customer', customerSchema)

File: CSharp/2. Managers/CSharp.Managers/ViewModels/Customer.cs
namespace CSharp.Managers.ViewModels
{
    public class Customer
    {
        public string FirstName { get; set; } = string.Empty;

        public string LastName { get; set; } = string.Empty;

        public int Age { get; set; } = 0;

        public Address Address { get; set; } = new Address();

        public string PhoneNumber { get; set; } = string.Empty;
    }
}
